content/news/release-20160508.md

```diff
@@ -0,0 +1,32 @@
+We are proud to announce a new stable release of OpenRA!
+
+This release is the result of 50 different developers, having produced a total of 1,225 commits since this cycle started six months ago. The code base has grown by more than 14,000 lines and almost 1,700 files have been touched during that time.
+
+So what has changed?
+
+The people among you that have already played the original Tiberian Dawn back in the day will notice that the Nod units in missions finally show up in their proper silver colors. And while we are on the topic of missions, you will also notice that the "Leave game" dialog now offers an option to restart the mission. This also works in single-player skirmish games, but unfortunately not for multi-player ones.
```

Comment:
I think this could be streamlined a bit:

```
So what has changed?

A lot of work went into fixing minor bugs and omissions to make the game experience more pleasant:

* Nod units in the Tiberian Dawn campaign finally use their original silver coloring.
* Starting a game after installing a map no longer causes the player to be kicked from the server.
* The lobby color validator will no longer complain as much as it used to.
* Walls now reliably stop bullets.
* Husks blocking refineries can now be targeted and destroyed.

We have also added the much-requested feature to restart missions and skirmish games, but this does not yet extend to multi-player games. Furthermore, the minimap radar will now also make use of team colors if those are enabled. Health bars can be set to only appear when the unit is damaged. Finally, the "Fragile Diplomacy" option has been removed.
```